src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,1327 @@ java -D&lt;property&gt;=&lt;value&gt;  \
+  -h, --help                Show this help message and exit.
+  -j, --javadocTree         This option is used to print the Parse Tree of the Javadoc comment. The
+                              file has to contain only Javadoc comment content excluding '/**' and
+                              '*/' at the beginning and at the end respectively. It can only be
+                              used on a single file and cannot be combined with other options.
+  -J, --treeWithJavadoc     This option is used to display the Abstract Syntax Tree (AST) with
+                              Javadoc nodes of the specified file. It can only be used on a single
+                              file and cannot be combined with other options.
+  -o=&lt;outputPath&gt;           Sets the output file. Defaults to stdout.
+  -p=&lt;propertiesFile&gt;       Sets the property files to load.
+  -s=&lt;suppressionLineColumnNumber&gt;
+                            Prints xpath suppressions at the file's line and column position.
+                              Argument is the line and column number (separated by a : ) in the
+                              file that the suppression should be generated for. The option cannot
+                              be used with other options and requires exactly one file to run on to
+                              be specified. Note that the generated result will have few queries,
+                              joined by pipe(|). Together they will match all AST nodes on
+                              specified line and column. You need to choose only one and recheck
+                              that it works. Usage of all of them is also ok, but might result in
+                              undesirable matching and suppress other issues.
+  -t, --tree                This option is used to display the Abstract Syntax Tree (AST) without
+                              any comments of the specified file. It can only be used on a single
+                              file and cannot be combined with other options.
+  -T, --treeWithComments    This option is used to display the Abstract Syntax Tree (AST) with
+                              comment nodes excluding Javadoc of the specified file. It can only be
+                              used on a single file and cannot be combined with other options.
+  -V, --version             Print version information and exit.
+  -w, --tabWidth=&lt;tabWidth&gt; Sets the length of the tab character.
+                                  Used only with -s option. Default value is 8.
+  -x, --exclude-regexp=&lt;excludeRegex&gt;
+                            Directory/file pattern to exclude from Checkstyle. Multiple excludes
+                              are allowed.
+  </code></pre>
+</div>
+
+<h4>
+18. Executing Ignored Modules (<span class="no-transform">-E</span>, --executeIgnoredModules)
+</h4>
+<p><strong>Goal:</strong> Run Checkstyle even on modules that are normally ignored.
```

Comment:
Run Checkstyle modules with ignore  severity, it helps to see all when Checker property 'severity' is set to 'ignore'
https://checkstyle.org/property_types.html#SeverityLevel